content/influxdb3/enterprise/visualize-data/superset.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Use Superset to visualize data
+list_title: Superset
+description: >
+  Install and run [Apache Superset](https://superset.apache.org/)
+  to query and visualize data stored in InfluxDB 3 Enterprise.
+weight: 101
+menu:
+  influxdb3_enterprise:
+    parent: Visualize data
+    name: Superset
+    identifier: query-with-superset
+influxdb3/enterprise/tags: [Flight client, query, flightsql, superset]
+metadata: [SQL only]
+alt_links:
+  cloud-serverless: /influxdb3/cloud-serverless/process-data/visualize/superset/
+  cloud-dedicated: /influxdb3/cloud-dedicated/process-data/visualize/superset/
+  clustered: /influxdb3/clustered/process-data/visualize/superset/
+draft: true 
```

Comment:
Why `draft`?

Reply:
I couldn't get Superset up and running. In it's own initialization script, it was writing Postgres 14-formatted data while trying to use Postgres 15. I guess the way these two versions store data is incompatible, so Superset wouldn't start. Didn't have time to troubleshoot, so I'm leaving it for future investigation.